### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by janko101**
### Changes proposed
- adds logo
- style

### CODE REVIEW

File: frontend/src/pages/auth/AuthPage.jsx [hunk 1]
```diff
@@ -27,8 +28,9 @@ export default function LoginPage() {
       <div className="container">
         <div className="row mt-4">
           <div className="col-md-6 " style={{ color: "white" }}>
-            <h1>SDA</h1>
-            <p>Starter template</p>
+
```
Comment (janko101) [inline frontend/src/pages/auth/AuthPage.jsx:31]: remove space

File: frontend/src/pages/home/HomePage.jsx [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
+        <img className="logo"src={logo}/>
         <p>
-          This starter template is based on Spring, PostgreSQL, React, React
-          router and Axios. Check the following links for documentation and
-          guides:
+         <em>Dumpty</em> is a luxury service of Garbage Collection.<br/>
+         Welcome to the world of upCycling.<br/>
+         Our garbage officers are the best in the world, they are all diplomed from the High University of Uppsala.
+
+
```
Comment (janko101) [inline frontend/src/pages/home/HomePage.jsx:20]: format code

File: frontend/src/pages/auth/AuthPage.jsx (hunk 1 above)
Comment (clecardona) [reply frontend/src/pages/auth/AuthPage.jsx:31]: you are right

File: frontend/src/pages/home/HomePage.jsx [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
           </li>
           <li>
-            <a href="https://github.com/axios/axios">Axios</a>
+            <a target="blank" href="https://en.wikipedia.org/wiki/Reference">References </a>
           </li>
+       
         </ul>
       </div>
+      
+      <img className="illustration"src={truck}/>
```
Comment (janko101) [inline frontend/src/pages/home/HomePage.jsx:33]: Example comment

Comment (janko101) [reply frontend/src/pages/home/HomePage.jsx:33]: Done

**Pull request merged**